mat/lu.go

```diff
@@ -277,26 +286,34 @@ func (lu *LU) LTo(dst *TriDense) *TriDense {
 }
 
 // UTo extracts the upper triangular matrix from an LU factorization.
-// If dst is nil, a new matrix is allocated. The resulting U matrix is returned.
-// UTo will panic if the receiver does not contain a factorization.
-func (lu *LU) UTo(dst *TriDense) *TriDense {
+//
+// If dst is empty, UTo will resize dst to be an upper-triangular n×n matrix.
+// If dst is non-empty, UTo will panic if dst is not n×n or if dst is not
+// Upper. UTo will also panic if the receiver does not contain a successful
```

Comment:
"When dst is non-empty, UTo will panic if dst is not n×n or not Upper."

Reply:
Done.